### ISSUE TITLE
a11y – focus

### ISSUE THREAD

**Issue by tiloyi** [adeo/mozaic-design-system#1592]
### I am opening an issue for

Styles (CSS, SCSS, ...)

### Package version

v1.74.0

### Description

Several components in the design system do not have a "focus" state, or have a "focus" state but do not behave in the right way.

A replay should be carried out to correct what can be corrected.

### What happened?

When users navigate using the keyboard, some components have no "focus" state or have a "focus" state that does not behave correctly.

Users have noticed this on components:
- DataTable Buttons
- Built-in Menu
- Password Input
- Stepper
- Tag
- Segmented Control

### What is expected?

Correct the behaviour of the "focus" state on these components.

### Is this a regression?

- [ ] Yes
- [X] No
- [ ] I don't know
